zqd/middleware.go

```diff
@@ -67,6 +68,25 @@ func accessLogMiddleware(logger *zap.Logger) mux.MiddlewareFunc {
 	}
 }
 
+func panicCatchMiddleware(logger *zap.Logger) mux.MiddlewareFunc {
+	logger = logger.Named("zqd")
```

Comment:
We need better facilities for this, but this logger should also use the `request-ID` value attached to context.